### ISSUE TITLE
Train/test split on objaverse

### ISSUE THREAD

**Issue by hzhupku** [liuyuan-pal/SyncDreamer#47]
Thanks for the great work! I am just wondering when u train the model on the objaverse dataset, did u use the whole dataset for training or have a train/test split like zero-123?

**Comment by liuyuan-pal** [comment]
Hi, since we re-render all the images, which makes it hard to follow the same train-test split as zero123. We use all the objaverse dataset for training.

**Comment by hzhupku** [comment]
Thanks for the reply!
